src/runtime/Makefile

```diff
@@ -587,20 +622,23 @@ USER_VARS += KERNELTYPE_ACRN
 USER_VARS += KERNELTYPE_CLH
 USER_VARS += KERNELPATH_ACRN
 USER_VARS += KERNELPATH
+USER_VARS += KERNELSEVPATH
 USER_VARS += KERNELTDXPATH
 USER_VARS += KERNELSEVPATH
 USER_VARS += KERNELSEPATH
+USER_VARS += KERNELSNPPATH
 USER_VARS += KERNELPATH_CLH
 USER_VARS += KERNELTDXPATH_CLH
 USER_VARS += KERNELPATH_FC
 USER_VARS += KERNELVIRTIOFSPATH
 USER_VARS += FIRMWAREPATH
+USER_VARS += FIRMWARESEVPATH
 USER_VARS += FIRMWARETDVFPATH
 USER_VARS += FIRMWAREVOLUMEPATH
 USER_VARS += TDSHIMFIRMWAREPATH
 USER_VARS += FIRMWARETDVFVOLUMEPATH
 USER_VARS += SEVFIRMWAREPATH
```

Comment:
We are adding `FIRMWARESEVPATH` above, but we still have this here and we are using `SEVFIRMWAREPATH` in the config file.

Reply:
Yeah, thanks for highlighting this as I forgot I meant to bring it up, we had `SEVFIRMWAREPATH := $(PREFIXDEPS)/share/ovmf/AMDSEV.fd` in `CCv0` and `FIRMWARESEVPATH := $(PREFIXDEPS)/share/ovmf/OVMF.fd` in `main`, so I wasn't sure if this was a case where `CCv0` should still be different as it does TEE specific logic, but if `FIRMWARESEVPATH := $(PREFIXDEPS)/share/ovmf/OVMF.fd` is correct I can switch the config to use it and tidy up `CCv0` a bit rather than keeping the alternative name around still?

Reply:
We also have a duplicate `SNPFIRMWAREPATH` and` FIRMWARESNPPATH` path. Those both point to `OVMF.fd`, which is correct.

Reply:
Cool, thanks. So I think the best thing is the value from `CCv0`, but the variable name from `main`, so `FIRMWARESEVPATH := $(PREFIXDEPS)/share/ovmf/AMDSEV.fd` Assuming the name change from `SEVFIRMWAREPATH` to `FIRMWARESEVPATH` in main wants to be kept?

Reply:
Yes, I think that is correct. Main consistently uses the naming scheme with the TEE type after the component name.

Reply:
Yeah, Fabiano started this when merging TDX, so I had the same troubles there! 😄